pkg/cmd/copytable/copytable.go

```diff
@@ -0,0 +1,34 @@
+package copytable
```

Comment:
From what I've read about project structures, the `pkg` subfolder is usually reserved for code meant to be imported in other projects. Also, I think you should follow cobra's project structure in this case (refer to: https://github.com/spf13/cobra#getting-started).

Reply:
that's the idea. we will need to do that for our private project